public-ui/src/app/data-browser/views/survey-chart/survey-chart-react.component.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { AgeChartReactComponent } from 'app/data-browser/charts/chart-age/chart-age-react.component';
+import { BioSexChartReactComponent } from 'app/data-browser/charts/chart-biosex/chart-biosex-react.component';
+import { VersionChartReactComponent } from 'app/data-browser/charts/chart-version/chart-version-react.component';
+import { TooltipReactComponent } from 'app/data-browser/components/tooltip/tooltip-react.component';
+import { ErrorMessageReactComponent } from 'app/data-browser/views/error-message/error-message-react.component';
+import { GraphType } from 'app/utils/enum-defs';
+import { triggerEvent } from 'app/utils/google_analytics';
+import * as React from 'react';
+
+interface State {
+    graphToShow: string;
+    selectedChartAnalysis: any;
+    displayGraphErrorMessage: boolean;
+    isLoaded: boolean;
+}
+
+interface Props {
+    graphButtons: any;
+    question: any;
+    answer: any;
+    selectedAnalysis: any;
+    selectedResult: any;
+    surveyName: any;
+    searchTerm: any;
+    surveyCountAnalysis: any;
+    isCopeSurvey: any;
+    versionAnalysis: any;
+}
+
+export class SurveyChartReactComponent extends React.Component<Props, State> {
+  constructor(props) {
+    super(props);
+    this.state = {graphToShow: this.props.isCopeSurvey ? GraphType.SurveyVersion : GraphType.BiologicalSex,
+    displayGraphErrorMessage: false, isLoaded: false, selectedChartAnalysis: null};
```

Comment:
Nit: for a multi-line object, put each property on its own line:
```suggestion
    this.state = {
      graphToShow: this.props.isCopeSurvey ? GraphType.SurveyVersion : GraphType.BiologicalSex,
      displayGraphErrorMessage: false,
      isLoaded: false,
      selectedChartAnalysis: null
    };
```

Reply:
changed thanks.